Cargo.toml

```diff
@@ -29,6 +29,14 @@ assets = [
 ]
 maintainer-scripts = "packaging/ubuntu"
 
+[profile.release]
```

Comment:
With the exception of `lto` and `panic`, all of these settings are the defaults for the release profile. See: https://doc.rust-lang.org/cargo/reference/manifest.html#the-profile-sections

Can you please explain why the `lto` and `panic` settings are set to these values?

Reply:
@pankajgoyal2 ^